src/flow-control/log-timings.js

```diff
@@ -0,0 +1,57 @@
+const Rx = require('rxjs');
+const _ = require('lodash');
+const { bufferCount, take } = require('rxjs/operators');
+const BaseHandler = require('./base-handler');
+
+module.exports = class LogTimings extends BaseHandler {
+    printExitInfoTimingTable(exitInfos) {
+        const exitInfoTable = _(exitInfos)
+            .map(({ command, timings, index, killed, exitCode }) => {
+                const readableDurationMs = (timings.endDate - timings.startDate).toLocaleString();
+                return {
+                    'call-index': index,
+                    name: command.name,
+                    duration: `~${readableDurationMs}ms`,
+                    'exit-code': exitCode,
+                    killed,
+                    command: command.command,
+                };
+            })
+            .sortBy('duration')
```

Comment:
Interesting, why sort by duration?

Reply:
I think if users are using the timing functionality they would be interested in the duration rankings ie knowing what was the fastest, slowest etc at a glance. Duration used to be a number but it wont sort properly as a string, I'll fix this